### PR TITLE
Added new icons 

### LINKED ISSUE #267: 3 new icons for DNA
1.
There is an error in an icon in the DNA.
The icon: "external link" must be replaced with the attachment icon with the same name.

2.
The "Article" and "Series Record" icons should be added to the DNA.

[3 icons.zip](https://github.com/youseedk/dna/files/5401773/3.icons.zip)

### PR DESCRIPTION
**Pull request by MichellDranig**
closes #267

### CODE REVIEW

**Review by razvandk — APPROVED** [review]

File: src/assets/svg/article.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (grandorf) [inline src/assets/svg/article.svg:1]: Let us remove width="40" height="40". In that way the icon will scale to the parent container and we don't need to override it with CSS.

File: src/assets/svg/external-link.svg [hunk 2]
```diff
@@ -1 +1,6 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><g fill="none" fill-rule="evenodd" stroke-linecap="round" stroke="currentColor" stroke-width="2"><path d="M31 19.73v9.732A3.538 3.538 0 0127.462 33H11.538A3.538 3.538 0 018 29.462V13.538A3.538 3.538 0 0111.538 10h9.731"/><path stroke-linejoin="round" d="M26 8h7v7"/></g></svg>
\ No newline at end of file
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (grandorf) [inline src/assets/svg/external-link.svg:1]: Let us remove width="40" height="40". In that way the icon will scale to the parent container and we don't need to override it with CSS.

File: src/assets/svg/player-series-record.svg [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (grandorf) [inline src/assets/svg/player-series-record.svg:1]: Let us remove width="40" height="40". In that way the icon will scale to the parent container and we don't need to override it with CSS.

**Review by grandorf — CHANGES_REQUESTED** [review]
Sorry I forgot to let tell you that we also remove width and height attributes on all SVG files. (The viewbox we still need).
By the way did your icons in the other PR also had witdh and height? Maybe we can remove those as well :-)

File: src/assets/svg/article.svg (hunk 1 above)
Comment (MichellDranig) [reply src/assets/svg/article.svg:1]: Will do! the other icons from friday also had width and height so I'll remove those as well :)

**Review by grandorf — APPROVED** [review]

**Pull request merged**